hardware/raspberrypi/gpio/gpio_pads_control.md

```diff
@@ -1,41 +1,10 @@
 # GPIO pads control
 
-### Address
+This document expains the current capabilities of the Raspberry Pi GPIO pins. It is applicable to all models up to an including the Mode 3B+. The main thing to note is that the GPIO drive strengths do not indicate a maximum current, but a maximum current under which the pad will still meet the specification. You should set the GPIO drive strengths to match the device being attached in order for the device to work correctly. 
```

Comment:
expains -> explains
an including -> and including
Mode 3B+ -> Model 3B+

:slightly_smiling_face: